test/gateway/eip20_gateway/confirm_redeem_intent.js

```diff
@@ -326,24 +326,6 @@ contract('EIP20Gateway.confirmRedeemIntent()', (accounts) => {
     );
   });
 
-  it('should fail when redeemer nonce is already consumed', async () => {
```

Comment:
Why did you delete this test? Shouldn't it still be tested? (e.g. try to use nonce 0 twice in a row)

Reply:
Similarly,
This test is already covered in `should fail to confirm redeem intent if its already confirmed once`.
In fact, this test was invalid as the redeemer nonce always started with 1.